__init__.py

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env python
```

Comment:
Please also add encoding to all your files:

# -*- coding: utf-8 -*-

Reply:
ditto but rember to escape the `*` or use a codeblock  because of markdown @degv364 : ` # -*- coding: utf-8 -*-`